feature-configs/cn-ran-overlays/README.md

```diff
@@ -0,0 +1,22 @@
+# cn-ran-overlays
+
+This folder contains an example configurations for 5G DU site
+(CU - work in progress)
+
+
+## Overview
+
+The [`all-in-one`](all-in-one) directory contains the Kustomize profile for deployment of DU integration features, namely:
+- SCTP MachineConfig patch
+- Performance addon operator and DU performance profile
+- PTP operator and slave profile
+- SR-IOV operator and associated profiles
```

Comment:
what about Local-Storage-Operator ?

Reply:
There is an EPIC for that in 4.8: CNF-1472 Local Persistent Volume Support (without dedicated storage node).  This can be updated under that work.